### ISSUE TITLE
Backup mysql creates infinite dumps

### ISSUE THREAD

**Issue by firecyberice** [stashed/stash#1030]
When creating a MySQL backup the session succeeds but the pod doing the backup exits with an error while updating the status. The backupsession is successful and the snapshot exists in the restic repository.  
```
NAME                        INVOKER-TYPE          INVOKER-NAME          PHASE       AGE
sample-mysql-backup-lpjtr   BackupConfiguration   sample-mysql-backup   Succeeded   14m
```
Instead of being in status Completed the pod is on Error state and gets restarted so endless new snapshots are created.
```
NAME                                           READY   STATUS    RESTARTS   AGE
stash-backup-sample-mysql-backup-lpjtr-dpnd2   0/1     Error     0          16m
stash-backup-sample-mysql-backup-lpjtr-8zlvb   0/1     Error     0          16m
stash-backup-sample-mysql-backup-lpjtr-xnmlq   0/1     Error     0          15m
stash-backup-sample-mysql-backup-lpjtr-gdjxz   0/1     Error     0          15m
stash-backup-sample-mysql-backup-lpjtr-4s2t6   0/1     Error     0          14m
stash-backup-sample-mysql-backup-lpjtr-xkdvw   0/1     Error     0          13m
stash-backup-sample-mysql-backup-lpjtr-r7d5v   0/1     Error     0          10m


```
Log of one of the `stash-backup` pods:
```
I0118 13:35:58.894926       1 status.go:69] Updating status of BackupSession: qa/sample-mysql-backup-lpjtr for host: host-0
Error: EOF
Usage:
  stash update-status [flags]

Flags:
      --backupsession string             Name of the Backup Session
  -h, --help                             help for update-status
      --kubeconfig string                Path to kubeconfig file with authorization information (the master location is set by the master flag).
      --master string                    The address of the Kubernetes API server (overrides any value in kubeconfig)
      --metrics-enabled                  Specify whether to export Prometheus metrics
      --metrics-labels strings           Labels to apply in exported metrics
      --metrics-pushgateway-url string   Pushgateway URL where the metrics will be pushed
      --namespace string                 Namespace of Backup/Restore Session (default "default")
      --output-dir string                Directory where output.json file will be written (keep empty if you don't need to write output in file)
      --prom-job-name string             Metrics job name (default "stash-prom-metrics")
      --repository string                Name of the Repository
      --restoresession string            Name of the Restore Session

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --bypass-validating-webhook-xray   if true, bypasses validating webhook xray checks
      --enable-analytics                 Send analytical events to Google Analytics (default true)
      --log-flush-frequency duration     Maximum number of seconds between log flushes (default 5s)
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files (default true)
      --service-name string              Stash service name. (default "stash-operator")
      --stderrthreshold severity         logs at or above this threshold go to stderr
      --use-kubeapiserver-fqdn-for-aks   if true, uses kube-apiserver FQDN for AKS cluster to workaround https://github.com/Azure/AKS/issues/522 (default true)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

F0118 13:35:58.981121       1 main.go:25] Error in Stash Main: EOF

```
I am using stash `v0.9.0-rc.2` and tried it with the mysql addon `mysql-5.7.25` and with `mysql.8.0.14`. If there is a new chart for v0.9.0-rc.3 i can test with this version as well.

Any Ideas?

**Comment by firecyberice** [comment]
ping @hossainemruz or  @tamalsaha
Do you have an idea how to fix this?

**Comment by hossainemruz** [comment]
Please wait for the next release. It should be very soon. Working on documentation update.

**Comment by madron** [comment]
I have the same problem with postgresql

**Comment by hossainemruz** [comment]
This should be fixed by `v0.9.0-rc.4`. Please, re-open if didn't.
